tools/screenshots/generate-integration-docs-screenshot

```diff
@@ -225,7 +226,7 @@ def capture_last_message_screenshot(bot: UserProfile, image_path: str) -> None:
         print(f"No message found for {bot.full_name}")
         return
     message_id = str(message.id)
-    screenshot_script = os.path.join(TOOLS_DIR, "message-screenshot.js")
+    screenshot_script = os.path.join(SCREENSHOTS_DIR, "message-screenshot.js")
```

Comment:
Never rename something without doing a `git grep` to see if it's referenced elsewhere:

```
$ git grep generate-integration-docs-scree
api_docs/incoming-webhooks-overview.md:  generated by running `tools/generate-integration-docs-screenshot --integration mywebhook`.
api_docs/incoming-webhooks-overview.md:  `tools/generate-integration-docs-screenshot --integration mywebhook`.
docs/documentation/integrations.md:  the screenshot using `tools/generate-integration-docs-screenshot`:
docs/documentation/integrations.md:  ./tools/generate-integration-docs-screenshot --integration integrationname
zerver/tests/test_integrations.py:            'and running "tools/generate-integration-docs-screenshot" to keep the screenshots up-to-date.'
```

You need to update those callers.

Reply:
updated